introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi, my name is Fredrik.
 My favorite CodeSchool Path is JavaScript.
+My deadly skills:
+* Git
+* JavaScript
+* PHP
+* HTML
+* CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?